state-chain/runtime/src/runtime_apis.rs

```diff
@@ -285,6 +285,7 @@ decl_runtime_apis!(
 		fn cf_swap_retry_delay_blocks() -> u32;
 		fn cf_swap_limits() -> SwapLimits;
 		fn cf_lp_events() -> Vec<pallet_cf_pools::Event<Runtime>>;
+		fn cf_minimum_chunk_size(asset: Asset) -> AssetAmount;
```

Comment:
This is a breaking change in the runtime interface. I'll raise an issue to handle this more gracefully (applies more generally, not just to this PR).